samples/Cylc/model.cylc

```diff
@@ -0,0 +1,138 @@
+#!Jinja2
```

Comment:
I think the CI failure might be to do with this line?

```
 Cylc has a sample with an interpreter (Jinja2) that isn't explicitly defined in languages.yml
```

This shebang line is a functional part of the language (configures Cylc to run the Jinja2 preprocessor on this file before ingestion), but can remove it if necessary.

Reply:
(The shebang doesn't affect tokenization of the file by the textmate grammar)

Reply:
Removed the Jinja2 shebangs as they are causing issues with CI.